Fixed the check for the scroll locker exists

benmcmahon100 opened the pull request:
Credit to @BerkeleyTrue for fining my stupid mistake!

benmcmahon100 committed:
Fixed the check for the scroll locker exists

Credit to @BerkeleyTrue for fining my stupid mistake!

BerkeleyTrue commented:
Hey @benmcmahon100 Thanks for the fix. Sorry we haven't been around to merge this yet. Things have been crazy the last week.

benmcmahon100 committed:
Update 2.0 to fix newly found issues

This should hopefully be the final time that the scroll-locker will need to be touched!

benmcmahon100 commented:
Ha Ha that's cool! So longs as my site breaking bug gets patched before deployment I'm happy! Also it gave me time to find another small issue that needing correcting. Thanks again for finding that original issue!
